pkg/server/settingswatcher/settings_watcher.go

```diff
@@ -435,8 +435,7 @@ func (s *SettingsWatcher) GetStorageClusterVersion() clusterversion.ClusterVersi
 		// settingswatcher, it means that the storage cluster has not yet been
 		// upgraded to 23.1. As a result, assume that storage cluster is at
 		// version 22.2.
-		// TODO(ajstorm): change this to 22.2 once the new version is minted.
-		storageClusterVersion := roachpb.Version{Major: 22, Minor: 1, Internal: 76}
+		storageClusterVersion := roachpb.Version{Major: 22, Minor: 2, Internal: 0}
```

Comment:
Ultra-nit: you might want to drop Internal altogether.

Reply:
i kind of like making it explicit so it's easier to tie it back to this line: https://github.com/cockroachdb/cockroach/blob/8e9b54896ce88d108da154c9134cab5f5c37625a/pkg/clusterversion/cockroach_versions.go#L522-L523